esp/src/eclwatch/SFDetailsWidget.js

```diff
@@ -151,6 +164,11 @@ define([
                 this.logicalFile.refresh();
             }
             this.subfilesGrid.startup();
+
+            if (this.logicalFile.isSuperfile === true && this.logicalFile.NumOfSubfiles > 1 && this.logicalFile.KeyType != "null") {
```

Comment:
```suggestion
            if (this.logicalFile.isSuperfile === true && this.logicalFile.NumOfSubfiles > 1 && this.logicalFile.KeyType !== "null") {
```

Reply:
Us "null" an esp response thing?

Reply:
null is returned if there is no value, however could use "", i dont believe it is esp specific, will test and fix

Reply:
Big difference between:
```
this.logicalFile.KeyType != "null"
```
and
```
this.logicalFile.KeyType !== ""
```
?

Reply:
so we expect 3 different key type values, if not returned, we get an undefined or "null" value, is there a better way to handle?

Reply:
So the possible KeyType values can be:
* undefined
* "null"
* "SomeOtherKeyType"
?

In which case wouldn't your logic need to be:
```
(this.logicalFile.isSuperfile === true && this.logicalFile.NumOfSubfiles > 1 && this.logicalFile.KeyType !== "null"&&this.logicalFile.KeyType !== undefined)
```
?
(assuming I understood correctly)

Reply:
should just be undefined, think i was just confusing myself on that one.